deepmd/pt/model/model/model.py

```diff
@@ -22,127 +22,117 @@ def __init__(self):
 
     def compute_or_load_stat(
         self,
-        fitting_param,
-        ntypes,
-        resuming=False,
         type_map=None,
         stat_file_dir=None,
         stat_file_path=None,
         sampled=None,
     ):
-        if fitting_param is None:
-            fitting_param = {}
-        if not resuming:
-            if sampled is not None:  # compute stat
-                for sys in sampled:
-                    for key in sys:
-                        if isinstance(sys[key], list):
-                            sys[key] = [item.to(env.DEVICE) for item in sys[key]]
-                        else:
-                            if sys[key] is not None:
-                                sys[key] = sys[key].to(env.DEVICE)
-                sumr, suma, sumn, sumr2, suma2 = self.descriptor.compute_input_stats(
-                    sampled
-                )
-
-                energy = [item["energy"] for item in sampled]
-                mixed_type = "real_natoms_vec" in sampled[0]
-                if mixed_type:
-                    input_natoms = [item["real_natoms_vec"] for item in sampled]
+        bias_atom_e = None
+        if sampled is not None:  # compute stat
+            for sys in sampled:
+                dict_to_device(sys)
+            sumr, suma, sumn, sumr2, suma2 = self.descriptor.compute_input_stats(
```

Comment:
not all the models have the data member `self.descriptor`, e.g. the pair_table atomic model. 
we should mv the method out of the base model. 

The same is for fitting. but not that the atomic_model may need the output stat.